lib/matplotlib/backend_bases.py

```diff
@@ -1363,12 +1354,12 @@ def __init__(self, name, canvas, guiEvent=None):
         self.guiEvent = guiEvent
 
 
+@cbook.deprecated("2.1")
```

Comment:
Can this deprecation warning have some more information? (eg. along the lines of 'idle_event is deprecated. Use the animations module instead.')

Would it also be worth putting in a what's new entry, saying this is deprecated and pointing people to what to use instead?